src/components/CampaignLocations/CampaignLocationsScreen.tsx

```diff
@@ -148,8 +150,24 @@ export const CampaignLocationsScreen: FunctionComponent<NavigationProps> = ({
               <AppText style={styles.selectCampaignHeader}>
                 Select campaign
               </AppText>
-              {Object.entries(authCredentials).map(
-                ([key, credentials], idx) => (
+              {Object.entries(authCredentials)
+                .sort(
+                  (
+                    a: [string, AuthCredentials],
+                    b: [string, AuthCredentials]
+                  ) => {
+                    const aText: string | undefined =
+                      allCampaignConfigs[a[0]]?.features?.campaignName;
+                    const bText: string | undefined =
+                      allCampaignConfigs[b[0]]?.features?.campaignName;
+                    if (!aText) return -1;
+                    else if (!bText) return 1;
+                    else if (aText.toLowerCase() > bText.toLowerCase())
+                      return 1;
+                    else return -1;
+                  }
+                )
```

Comment:
I noticed there are 2 things we're doing here, first getting the campaign name for each `authCredential`, then sorting.

I suggest we do the following in succession:
1. `map` first to create an array of `AuthCredentials & { name: string }`,
2. Then  `sort`.
3. Then `map` again to create the actual `CampaignLocationsListItem`components

For the sort, we can make use of lodash's sortBy: https://lodash.com/docs/4.17.15#sortBy
This will keep the code cleaner (:

Reply:
committed with lodash